src/App/Pages/Accounts/LoginSsoPageViewModel.cs

```diff
@@ -16,6 +17,8 @@ namespace Bit.App.Pages
 {
     public class LoginSsoPageViewModel : BaseViewModel
     {
+
+        private const string RedirectUri = "bitwarden://sso-callback";
```

Comment:
consts should be upper case => `REDIRECT_URI` and don't leave an empty line above I'd separate with a new line the const from the fields